tests/gflownet/envs/test_htorus.py

```diff
@@ -23,7 +23,3 @@ def env():
 )
 def test__get_action_space__returns_expected(env, action_space):
     assert set(action_space) == set(env.action_space)
-
-
-def test__continuous_env_common(env):
-    return common.test__continuous_env_common(env)
```

Comment:
The development of this environment has been discontinued for quite a while, so many things don't work as they should.

Reply:
Since those are some of the most basic tests that don't work as they should... does that we mean that we simply shouldn't use that environment? Even going as far as deprecating/removing this environment?

Reply:
Yes.